src/factories/request.js

```diff
@@ -65,14 +66,18 @@ class RequestFactory {
         .catch(error => reject(error))
     })
 
-    promise.then(response => {
-      const credentials = new Credentials(
-        config.client_id,
-        response.access_token,
-        response.expires
-      )
-      storage.set('moltinCredentials', JSON.stringify(credentials))
-    })
+    promise
+      .then(response => {
+        const credentials = new Credentials(
+          config.client_id,
+          response.access_token,
+          response.expires
+        )
+        storage.set('moltinCredentials', JSON.stringify(credentials))
+      })
+      .catch(() => {
+        storage.delete('moltinCredentials')
```

Comment:
@agrohs Could you remove this? We can then apply this in a separate PR.

Reply:
@notrab, shouldn't remove the catch itself as it is throwing an `UnhandledPromiseRejection` when a bad fetch is passed in (as tested in the unit test), but can remove the `storage.delete('moltinCredentials')` inside and you guys can figure out whether that is a desired fix/feature to add in on the moltin end at some point. Will update commit now w/ the empty catch block now per our discussion.